package-manifest.toml

```diff
@@ -134,10 +134,34 @@ commit = "2be097ddd1d3fd8e7f56bc0a4bfd696253b11454"
 # https://buildomat.eng.oxide.computer/public/file/oxidecomputer/maghemite/image/<commit>/mg-ddm.sha256.txt
 sha256 = "94218915ec6fed75dcc81d736bd5e9ef62c9eb651a67a943b297d94d6b390941"
 
-[external_package.dendrite]
+[external_package.dendrite_tofino_stub]
 service_name = "dendrite"
 zone = true
-[external_package.dendrite.source]
+[external_package.dendrite_tofino_stub.only_for_targets]
+switch_variant = "tofino_stub"
```

Comment:
The git diff made this look worse than it actually is, but this is the crux of the change.